### PR TITLE
Updated to new API

### PR DESCRIPTION
**Pull request by EazyFTW**
I might have forgotten something, let me know if it doesn't work.

### CODE REVIEW

**Comment by sk89q** [comment]
Looks pretty good. Will be three weeks before I'll get to my computer though

File: launcher/src/main/java/com/skcraft/launcher/Launcher.java [hunk 1]
```diff
@@ -366,6 +366,36 @@ public URL propUrl(String key, String... args) {
         return HttpRequest.url(prop(key, args));
     }
 
+    public static URL getMetaURL(String version) throws IOException, InterruptedException {
+        URL url = new URL("https://launchermeta.mojang.com/mc/game/version_manifest.json");
+        LauncherJSON launcherJSON = HttpRequest
```
Comment (i9hdkill) [inline launcher/src/main/java/com/skcraft/launcher/Launcher.java]: Imports for new classes are missing

**Comment by i9hdkill** [comment]
Plus, the builder needs an update aswell:
https://github.com/SKCraft/Launcher/blob/40b5fb838c1af16812653e002fe851a64696cced/launcher-builder/src/main/java/com/skcraft/launcher/builder/PackageBuilder.java#L291

**Comment by cmspeedau** [comment]
Installing forge-1.14.4-28.2.4-installer.jar...
[warning] Task failed
java.lang.NullPointerException
	at com.skcraft.launcher.builder.PackageBuilder.processLoader(PackageBuilder.java:162)
	at com.skcraft.launcher.builder.PackageBuilder.addLoaders(PackageBuilder.java:128)
	at com.skcraft.launcher.builder.PackageBuilder.main(PackageBuilder.java:392)
	at com.skcraft.launcher.creator.controller.task.PackBuilder.call(PackBuilder.java:66)
	at com.skcraft.launcher.creator.controller.task.PackBuilder.call(PackBuilder.java:21)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)

in the build launcher i get this error using barpec12 release and 1.14.4 mc dose load but im not sure how to get this to him

**Comment by hedgehog1029** [comment]
Superseded by changes on master, cheers.

**Pull request closed without merging**